Tests/Acceptance/PaypalCheckoutTest.php

```diff
@@ -16,6 +16,9 @@
  */
 class PaypalCheckoutTest extends CheckoutTestCase
 {
+    // catch possible A/B tests
+    protected $retryTimes = 1;
```

Comment:
Where is this used?

Reply:
It's used in the base class: https://github.com/OXID-eSales/testing_library/blob/master/library/AcceptanceTestCase.php#L24